pkg/app/carbonapi/app_test.go

```diff
@@ -155,6 +155,9 @@ func SetUpTestConfig() (*App, http.Handler) {
 
 	setUpConfig(app, logger)
 	handler := initHandlers(app, logger)
+
+	go ProcessRequests(app)
```

Comment:
I think this is not supposed to be run in a new goroutine although it'll work properly.

Reply:
Yes, you're right. This has to be corrected. I'll correct this in the next PR as it has no impact.